web-client/src/presenter/actions/StatusReportOrder/prepareStatusReportOrderAction.ts

```diff
@@ -34,11 +35,19 @@ export const prepareStatusReportOrderAction = ({
     .getUtilities()
     .formatDateString(statusReportFilingDate, FORMATS.MONTH_DAY_YEAR);
 
+  let calendaredLine = ''
+  if(isCalendared) {
+    const formattedTrialDate = applicationContext
+      .getUtilities()
+      .formatDateString(caseDetail.trialDate, FORMATS.MONTH_DAY_YEAR);
+    calendaredLine = `This case is set for trial at the session of the Court commencing on ${formattedTrialDate} in ${caseDetail.trialLocation}.`;
+  }
+
   const filedLine =
     isLeadCase &&
     issueOrder === STATUS_REPORT_ORDER_OPTIONS.issueOrderOptions.allCasesInGroup
-      ? `<p class="indent-paragraph">On ${statusReportFilingDateFormatted}, a status report was filed (Lead case document no. ${statusReportIndex}). For cause, it is</p>`
-      : `<p class="indent-paragraph">On ${statusReportFilingDateFormatted}, a status report was filed (Document no. ${statusReportIndex}). For cause, it is</p>`;
+      ? `<p class="indent-paragraph">${calendaredLine} On ${statusReportFilingDateFormatted}, a status report was filed (Lead case document no. ${statusReportIndex}). For cause, it is</p>`
```

Comment:
Total nitpick, but I'd recommend putting the space in `calendaredLine` itself, or incorporating it here with a ternary (e.g., calendaredLine ? etc.) instead of after it. (If there were ever some weird styling change, we might end up rendering the initial space if `calendaredLine` were empty.)